packages/x-components/src/components/layouts/single-column-layout.vue

```diff
@@ -68,11 +68,13 @@
 
```

Comment:
move the tokens form `packages/x-components/src/design-system/components/layout/columns.tokens.scss`
to here, and remove those that are not necessary as it can be done by the slots content.

Reply:
Also document them with the use and default value